README.md

```diff
@@ -1 +1,39 @@
-# Portfolio-setup-and-mobile-version-skeleton
\ No newline at end of file
+# Portfolio-setup-and-mobile-version-skeleton
+ Description of the project: 
+    This project represents the mobile version of my portfolio where all my projects and skills will be uploaded on it in the future.
+
+
+## Built With
+
+- Major languages: HTML, CSS
+- linter for code analysis
+
+
+## Authors
+
+👤 **Mohammed Elseddik Rebbouh**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Microverse team 
+- My coding partner
+- My standup team
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] The same thing for the MIT license link. I would recommend fixing it to make README professional.